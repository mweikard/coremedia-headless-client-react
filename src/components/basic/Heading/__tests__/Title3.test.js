// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Title3 } from '..';

describe('Title3 Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Title3>Content</Title3>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
