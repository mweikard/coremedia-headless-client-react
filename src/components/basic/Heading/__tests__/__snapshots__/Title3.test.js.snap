// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title3 Component should render correctly 1`] = `
.c0 {
  font-family: Helvetica,Arial,sans-serif;
  font-size: 4rem;
  font-weight: 300;
  text-transform: uppercase;
  line-height: 1.4em;
  color: #000;
}

<h3
  className="c0"
>
  Content
</h3>
`;
